presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -72,8 +74,13 @@ public PlanBuilder(PlanNodeIdAllocator idAllocator)
 
     public ValuesNode values(Symbol... columns)
     {
+        return values(idAllocator.getNextId(), columns);
+    }
+
+    public ValuesNode values(PlanNodeId id, Symbol... columns)
```

Comment:
so you know what id to map the stats to.  I couldn't think of a better way to do that.